src/main/webapp/resources/js/apis/files/files.js

```diff
@@ -0,0 +1,132 @@
+import axios from "axios";
```

Comment:
This file should only contain calls to the the server to either get or post data to the server. Most of this code should be in a react component no?